### PR TITLE
Fix/modbus-read-write

### PR DESCRIPTION
**Pull request by Shiny380**
re-worked modbus polling and modbus point fields
**breaking DB changes!**
- moved `write_value` to base point model
  
- some modbus point fields are auto changed or now have defaults
- trying to read/write to a point delete is moved and handled elsewhere by try/excepts
- general cleanup and fixes for modbus polling
- `zero_mode` change to `zero_based` and implemented

### CODE REVIEW

File: src/source_drivers/modbus/services/rtu_polling.py [hunk 1]
```diff
@@ -55,9 +55,7 @@ def __poll(self):
         results = self.__get_all_points()
         # TODO: separate thread for each network
         for network, device, point in results:
-            if all(v is not None for v in results):
-                time.sleep(RtuPolling._polling_period)
-                poll_point(self, network, device, point, ModbusType.RTU)
+            poll_point(self, network, device, point, ModbusType.RTU)
```
Comment (RaiBnod) [inline src/source_drivers/modbus/services/rtu_polling.py:58]: May be we get issue like previously we had so we were checking all these should be not `None`. But it's fine to leave this as is for now.

**Review by RaiBnod — APPROVED** [review]

File: src/source_drivers/modbus/models/network.py [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
-    device_timeout_global = db.Column(db.Float(), nullable=False)
-    point_timeout_global = db.Column(db.Float(), nullable=False)
-    rtu_port = db.Column(db.String(80))
-    rtu_speed = db.Column(db.Integer())
-    rtu_stop_bits = db.Column(db.Integer())
-    rtu_parity = db.Column(db.Enum(ModbusRtuParity))
+    timeout = db.Column(db.Float(), nullable=False, default=1)
+    device_timeout_global = db.Column(db.Float(), nullable=False, default=1)
+    point_timeout_global = db.Column(db.Float(), nullable=False, default=1)
+    rtu_port = db.Column(db.String(80), nullable=False)
```
Comment (RaiBnod) [inline src/source_drivers/modbus/models/network.py:14]: Make it nullable=True, we don't need this on TCP and we have it's custom validation.

**Review by RaiBnod — CHANGES_REQUESTED** [review]

File: src/source_drivers/modbus/services/rtu_polling.py (hunk 1 above)
Comment (Shiny380) [reply src/source_drivers/modbus/services/rtu_polling.py:58]: I do try/excepts in `poll.py` instead.
tested by trying to read and update to deleted points after already fetched in loop and catches fine

File: src/source_drivers/modbus/models/network.py (hunk 2 above)
Comment (Shiny380) [reply src/source_drivers/modbus/models/network.py:14]: truuuuuuueeeeeeee! nice catch

**Pull request merged**